package.xml

```diff
@@ -6,6 +6,10 @@
   <description>The package provides GoogleMock.</description>
   <maintainer email="dthomas@osrfoundation.org">Dirk Thomas</maintainer>
   <license>BSD</license>
+  
+  <buildtool_depend>cmake</buildtool_depend>
+  
+  <buildtool_export_depend>cmake</buildtool_export_depend>
```

Comment:
Why does it need to export this dependency? A downstream packages could try to use it but use a different build system.